olexandr.vakulinskiy/HW3/Task1/prime_numbers.rb

```diff
@@ -0,0 +1,54 @@
+require 'matrix'
+class Matrix
```

Comment:
U can, but I don't recommend adding methods to Ruby classes. In the future, it is better to create your own and work with it. You can [inherit your class](https://www.rubyguides.com/2019/01/what-is-inheritance-in-ruby/) from Matrix and do with it what u want 🙂